lib/check/single_module_file.ex

```diff
@@ -0,0 +1,59 @@
+defmodule CompassCredoPlugin.Check.SingleModuleFile do
+  use Credo.Check,
+    base_priority: :high,
```

Comment:
Is it a really high priority? If yes, how do we determine the priority level?

Other our implemented checks are low priority 🤔

Reply:
I think the priority: `high` is the thing `do` or `don't`. `low/normal` is something like `prefer` or `not prefer`. I also checked the [credo docs](https://hexdocs.pm/credo/Credo.Check.html) and don't see they mention about how we will define the priority for a particular issue. 😄 

What do you think @Nihisil @liamstevens111 @hanam1ni ?

Reply:
For this check feels high priority to me because it is not just the code styling but relates to the code structure of how devs create and separate modules.

Also, the way to determine the priority that Thanh mentioned sounds reasonable to me. Anyway, I think we need more input from other devs about this.

Reply:
I think I left mine as `medium` (I copied Alex or Micky's 😆 ) so yeah it's artibary, I don't think it's a big deal for now but I think this check probably has higher priority than say my `single line do` readability one 👍

Reply:
Ok, I think for now it is good to keep it as hight priority